dmscripts/generate_framework_agreement_signature_pages.py

```diff
@@ -65,3 +66,5 @@ def render_pdf_for_each_html_page(html_pages, html_dir, pdf_dir):
         exit_code = subprocess.call(['wkhtmltopdf', 'file://{}'.format(html_path), pdf_path])
         if exit_code > 0:
             logger.error("ERROR {} on {}".format(exit_code, html_page))
+            ok = False
```

Comment:
Yeah I'm not sure.  Feels like it's worth trying the rest and then it'll be more obvious looking at the output whether it's an issue with just one file or a general issue with everything.